torch/library.py

```diff
@@ -171,10 +172,27 @@ def impl(self, op_name, fn, dispatch_key=''):
         self._op_impls.add(key)
 
     def _destroy(self):
+        if self.m is not None:
+            self.m.reset()
```

Comment:
Do we also need to call reset on `m` in _del_library?

Reply:
I'm not sure, it might be valid for someone (us, somewhere in PyTorch) to grab self.m and manage it. The goal of this PR is to offer a way to manually clear the library.